src/components/form/InputPath.js

```diff
@@ -16,12 +16,12 @@ export default class InputFilename extends Component {
     if (type == 'posts') {
       const date = moment().format('YYYY-MM-DD');
       placeholder = `${date}-your-title.md`;
-    }else if (type == 'datafiles') {
-      placeholder = 'your-filename.yml';
+    }else if (type == 'data files') {
+      placeholder = '<directory/>your-filename.yml';
```

Comment:
actually, it was intended to show that `directory` and `/` are optional. The `/` needed only if there was a directory param.
```
(directory/)?filename.yml
(directory/)filename.yml
[directory/]filename.yml
<directory/>filename.yml
```